paper.md

```diff
@@ -1,33 +1,36 @@
 ---
-title: 'Separation using ChatGPT'
+title: 'Zero-Shot Duet Singing Voices Separation with Diffusion Models'
 tags:
-  - separation
-  - u-net
+  - diffusion models
+  - singing voice separation
+  - inverse problems
+  - zero-shot learning
 authors:
-  - name: Fabian-Robert Stoter^[co-first author] # note this makes a footnote saying 'co-first author'
-    orcid: 0000-0003-0872-7098
-    affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author Without ORCID^[co-first author] # note this makes a footnote saying 'co-first author'
+  - name: Chin-Yun Yu
+    orcid: 0000-0003-3782-2713
+    affiliation: 1
+  - name: Emilian Postolache
     affiliation: 2
-  - name: Author with no affiliation^[corresponding author]
-    affiliation: 3
+  - name: Emanuele Rodolà
+    affiliation: 2
+  - name: György Fazekas
+    affiliation: 1
 affiliations:
- - name: Audioshake
+ - name: Queen Mary University of London
    index: 1
- - name: Institution Name
+ - name: Sapienza University of Rome
    index: 2
- - name: Independent Researcher
-   index: 3
-date: 10 September 2023
+date: 9 October 2023
 bibliography: paper.bib
 arxiv-doi: 10.21105/joss.01667
 ---
 
 # Abstract
 
-The abstract should be around 250 words long as usual, and should be provided
-for submissions of all categories: posters, long talks and discussions.
-
-* Please briefly describe the particular points your submission focuses on and/or the
-problems it aims to solve.
-* For all categories except discussions, please also briefly summarize the contributions.
+In recent studies, diffusion models have shown promise as priors for solving audio inverse problems [@moliner2023solving; @saito2023unsupervised; @yu2023conditioning; @murata2023gibbsddrm], including source separation [@mariani2023multi]. 
+These models allow us to sample from the posterior distribution of a target signal given an observed signal by manipulating the diffusion process.
+However, when separating audio sources of the same type, such as duet singing voices, the constraints learned by the diffusion process may not be sufficient to preserve the identity of the target source, unless additional guidance is provided.
+Tackling this problem will be useful for separation sources in choir music, or a mixture of multiple instruments with similar timbre, without acquiring large amounts of paired data.
+In this paper, we examine this problem in the context of duet singing voices separation, and propose a method to enforce the identity of the target voice by splitting the mixture into overlapping segments and performing posterior sampling in an autoregressive manner.
+We evaluate the proposed method on the MedleyVox dataset [@jeon2023medleyvox] with different overlap ratios, and show that the proposed method outperforms the baseline with naive posterior sampling.
```

Comment:
Is there a statement you can make already regarding a comparison against the medleyvox sepformer baselines?

Reply:
We didn't compare our approach with the one proposed in medleyvox as our method is completely unsupervised.  We compare different configurations of conditional sampling in an unconditional singing voice diffusion model. We plan to add other unsupervised baselines if time allows.